perfmetrics/scripts/fetch_metrics.py

```diff
@@ -40,14 +71,15 @@
     end_time_sec = job[fio_metrics.consts.END_TIME]
 
     # Print start and end time of jobs
-    print("Start time: ",start_time_sec)
-    print("End time: ",end_time_sec)
+    print("Start time: ", start_time_sec)
+    print("End time: ", end_time_sec)
 
     rw = job[fio_metrics.consts.PARAMS][fio_metrics.consts.RW]
-    print(f'Getting VM metrics for job at index {ind+1}...')
-    metrics_data = vm_metrics_obj.fetch_metrics(start_time_sec, end_time_sec, INSTANCE, PERIOD_SEC, rw)
+    print(f'Getting VM metrics for job at index {ind + 1}...')
+    metrics_data = vm_metrics_obj.fetch_metrics(start_time_sec, end_time_sec,
```

Comment:
This formatting changed happened automatically? If not please revert back.

Reply:
Yes it is automatically happened.